lib/stripe.rb

```diff
@@ -220,10 +220,8 @@ def self.uri_encode(params)
   end
   class << self
     extend Gem::Deprecate
-    deprecate :uri_encode, "Stripe::Util#encode_parameters", 2016, 01
+    deprecate :uri_encode, "Stripe::Util#encode_parameters", 2016, 0o1
```

Comment:
Ah-ha! I didn't know that Ruby used `0` as the octal prefix. That clearly wasn't the intention here so we'll need to track those instances and remove the prefixes.

Reply:
Wow, Rubocop's off to a great start with finding problems for us!